src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
```

Comment:
Consider using `const` for `totalCost` since it's not reassigned but only modified in the subsequent `if` blocks. This adheres to the checklist item 3 preferring `const` over `let` where possible to signal that the variable should not be completely reassigned.